src/module/components/character-generator/character-generator.js

```diff
@@ -260,14 +260,29 @@ export class ForbiddenLandsCharacterGenerator extends Application {
 			}
 		}
 		character.childhood = this.rollOn(kin.childhood);
+		character.childhood.attributes = this.mappingAttributes(character);
+		character.childhood.skills = this.mappingSkills(character.childhood.skills);
```

Comment:
This part needs to be added to `handleRollChildhood()` and `handleInputChildhood`.